frontend/src/views/WheresMyGene/components/Filters/index.tsx

```diff
@@ -241,13 +240,15 @@ export default memo(function Filters({ isLoading }: Props): JSX.Element {
   );
 
   function TooltipWrapper({ children }: { children: ReactElement }) {
-    const Wrapper = areFiltersDisabled ? Tooltip : Fragment;
+    if (areFiltersDisabled) {
+      return (
+        <Tooltip title="Please select an organism, tissue and at least one gene to use these filters.">
+          {children}
+        </Tooltip>
+      );
+    }
 
-    return (
-      <Wrapper title="Please select an organism, tissue and at least one gene to use these filters.">
```

Comment:
This triggers React warning that `title` attr is not supposed to be passed to `Fragment`